src/Vega4Wrapper.js

```diff
@@ -0,0 +1,207 @@
+var makeValidator = require('domain-validator');
+
+class Vega4Wrapper {
+    /**
+     * Shared library to wrap around vega code
+     * @param {Object} wrapperOpts Configuration options
+     * @param {Object} wrapperOpts.loader Vega-loader object, its sanitize(), file() and load() method will be overwrited
+     * @param {Object} wrapperOpts.domains allowed protocols and a list of their domains
+     * @param {Object} wrapperOpts.domainMap domain remapping
+     * @param {Function} wrapperOpts.logger
+     * @param {Function} wrapperOpts.formatUrl
+     * @param {string} [wrapperOpts.languageCode]
+     * @constructor
+     */
+    constructor(wrapperOpts) {
+        var self = this;
```

Comment:
for the newer code, don't use `var`.  Use `const` or `let` (if value changes).  Also, one declaration per line (at some point we will need to add eslint, but that can wait)